app/helpers/github.rb

```diff
@@ -6,17 +6,36 @@ def github_create_webhook(submission)
     client = Octokit::Client.new access_token: current_user.github_access_token
     hooks = client.hooks(repo_full_name)
 
-    if !hooks || hooks.empty? || !hooks.map{|x| x.events}.any?{|e| e.include? "pull_request"}
```

Comment:
Look like it meant something like don't create a new hook if there was at least one with `"pull_request"` event.